docs/release-notes.md

```diff
@@ -32,6 +32,19 @@ updates as well.
   2.9.10 has an infinite loop in a certain end-of-file situation." Fix
   by updating a dependency in documentation generation.
 
+### Zephyr-RTOS compatibility
+- for compatibility with zephyr 2.2.0 need to revert commits which align MCUBoot
+  to the zephyr master branch at the release time. These commits are reverted on
+  1.6.0-revert-zephyr-2.2.0-compatible branch.
+
+- Zephyr-RTOS references (by git SHA or git tag) a compatible MCUBoot version in
+  its manifest.
+  You might use any of zephyr running-master version which references
```

Comment:
I'm not sure this is quite right. If someone is using the Zephyr manifest, they will get Zephyr's fork of MCUboot, which will work with Zephyr. I think we should just leave this as a mention of a particular commit this works with (and probably not even 'after'.